notm-specs/src/test/groovy/mockapis/Elite2Api.groovy

```diff
@@ -220,6 +220,7 @@ class Elite2Api extends WireMockRule {
       get(urlMatching("/api/bookings/.+/iepSummary"))
         .willReturn(aResponse()
         .withStatus(200)
+        .withHeader('Content-Type', 'application/json')
```

Comment:
superagent a bit more strict, if json header not returned then won't automatically parse into json